### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by insideoutclub**
Thanks for putting this together. I learned some things while going through the exercises.

### CODE REVIEW

File: exercises/00_welcome/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Chapter 0: References and Lifetimes Revision
```
Comment (tfpk) [inline exercises/00_welcome/README.md:1]: Curious if there's a reason to change "Revision" to "Review". Was this confusing to you?

Comment (insideoutclub) [reply exercises/00_welcome/README.md:1]: Yes, to "revise" something is to make changes to something to improve it. To "review" something in this context is to study again a topic that you've already learned in the past. Here's a good link:

https://www.espressoenglish.net/difference-between-review-and-revise/

Here's some relevant text from that link:
How To Use The Word REVIEW In English
All right, let’s begin with the word review. It has a couple different uses:

One is when you take something you previously learned and you study it again.

Comment (tfpk) [reply exercises/00_welcome/README.md:1]: Ah! That explains it: I speak Australian (i.e. British) English, and:

> However, in British English, the word revise is also used for studying something again. In American English we would always say “review for a text/exam” and in British English you could say “revise for a text/exam.” Again, that use of revise to mean “study again” is specific to British English.

Happy to change it, I try and target American English online. Thanks, and TIL.

Comment (insideoutclub) [reply exercises/00_welcome/README.md:1]: I wasn't aware of this use of "revise" in British English. I learned something today as well. Cheers!

**Comment by insideoutclub** [comment]
I messed up the reference from chapter 7 to chapter 10. Do you know how to fix it?

**Comment by tfpk** [comment]
I'll have a look...

**Comment by tfpk** [comment]
@insideoutclub fixed.

**Pull request merged**